### PR TITLE
test: RHOAM 1.4.0 test cases

### PR DESCRIPTION
**Pull request by psturc**
# Description
Test case for https://github.com/integr8ly/integreatly-operator/pull/1652

### CODE REVIEW

**Comment by psturc** [comment]
Hi @KevFan, I created a test case based on your fix, would you mind giving it a look and add `/lgtm` if it looks ok? Thanks

File: test-cases/tests/installation/a35-3scale-deploymentconfig-should-rescale-if-it-failed-during-i.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+2. Manually cause system-app DC to fail
+
+```bash
+oc rollout latest dc/system-app -n redhat-rhoam-3scale
+sleep 10
+PRE_HOOK_POD=$(oc get pod -n redhat-rhoam-3scale -o name | grep "hook-pre" | tail -n1)
+oc delete $PRE_HOOK_POD -n redhat-rhoam-3scale
+```
+
+3. Verify system-app-deploy pod is an error state
```
Comment (KevFan) [inline test-cases/tests/installation/a35-3scale-deploymentconfig-should-rescale-if-it-failed-during-i.md:38]:
```suggestion
3. Verify system-app-deploy pod is in an error state
```

File: test-cases/tests/products/h26-3scale-users-permissions-do-not-change-after-changing-userna.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+tags:
+  - manual-selection
+---
+
+# H26 - 3scale user's permissions do not change after changing username in 3scale UI
+
+## Description
+
+When user modifies a username in 3scale UI, the new username gets deleted and original name recreated after integreatly-operator reconciles
+Verify that the original user permissions (admin) are not changed after the username gets recreated
```
Comment (KevFan) [inline test-cases/tests/products/h26-3scale-users-permissions-do-not-change-after-changing-userna.md:19]:
```suggestion
When user modifies a username in 3scale UI, the new username gets deleted and original name recreated after integreatly-operator reconciles.
Verify that the original user permissions (admin) are not changed after the username gets recreated
```
Add `.` as sentences will merge when markdown is viewed

**Review by KevFan — COMMENTED** [review]
Look good :+1: 

Just have small comments on typos

**Comment by psturc** [comment]
@KevFan  thanks for your review 👍

**Comment by KevFan** [comment]
Test cases looks good to me :+1: 
/lgtm

**Pull request merged**